### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by dewslyse**
This is the second milestone for the Math Magicians application.
In this milestone, I developed the UI for the calculator app. 
Specifically, I:
- deleted React boilerplate styles and images
- created a `components` directory containing the files `Calculator.js` and `Calculator.scss`
- used class based components for the calculator UI and imported it into the main component (`App.js`)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (Kingstalux) [inline src/App.js]: - [x] Great job so far 👍 Kindly make your `App.js` to be class-based so as to follow the requirements 💯

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @dewslyse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

149637855-7722d04c-b45d-4185-925f-7de047106461

## Highlights
✅ App displays well 🔥 
✅ Good pull request description 👏 
✅ Good commit messages 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (dewslyse) [reply src/App.js]: @Kingstalux,  Thanks for the review. Changes effected.

**Review by EricMbouwe — APPROVED** [review]
### Hi @dewslyse, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**